scripts/lintr.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd ../shinydashboard/lantern
+echo "Running R lintr"
```

Comment:
I was able to install the lntr r package via this bash command. Try putting this into the script -- if it works then some of the new README text will be outdated
`echo 'install.packages("lintr", dependencies = TRUE, repos="http://cran.rstudio.com/")' | R --save`

Reply:
All set!